### ISSUE TITLE
TransferFunction is not a Control.LTISystem

### ISSUE THREAD

**Issue by baggepinnen** [mfalt/Control.jl#14]
I find this behavior unexpected. The type TransferFunction is a subsytpe of LTISystem, but an instance of TF is not an LTISystem. An instance of Float64 is however a Real
```julia
julia> typeof(P)
Control.TransferFunction

julia> isa(P, Control.LTISystem)
false

julia> Control.TransferFunction <: Control.LTISystem
true

julia> isa(1.0, Real)
true

```

**Comment by mfalt** [comment]
I am not able to reproduce this, I get
```julia
julia> using Control
julia> P = tf(1)
TransferFunction:
1.0
---
1.0

Continuous-time transfer function model

julia> typeof(P)
Control.TransferFunction

julia> isa(P, Control.LTISystem)
true
```

**Comment by mfalt** [comment]
Fixed in bf0504a
